src/main/java/BasketInformations.java

```diff
@@ -0,0 +1,98 @@
+package com.company;
+
+import java.util.HashMap;
+
+/**
+ * This class is a basket
+ */
+public class BasketInformations {
+
```

Comment:
I suggest to use singular for the name of the class, perhaps even the "Information" part could be removed.
See more: 
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html